docs/sql/index.md

```diff
@@ -282,7 +327,7 @@ literal = INTEGER | FLOAT | STRING | HEX | TRUE | FALSE ;
 ```
 
 The following describes how to construct literal values,
-for [SATS] data types, in Spacetime SQL.
+for SATS data types, in Spacetime SQL.
```

Comment:
Commas aren't necessary here - "for SATS data types" is not a subordinate clause. GH won't let me suggest, but should be:

```
The following describes how to construct literal values for SATS data types in Spacetime SQL.
```